src/bp/core/services/ghost/disk-driver.ts

```diff
@@ -56,7 +56,11 @@ export default class DiskStorageDriver implements StorageDriver {
 
   async deleteDir(dirPath: string): Promise<void> {
     try {
-      return fse.removeSync(this.resolvePath(dirPath))
+      const path = this.resolvePath(dirPath)
+      setTimeout(() => {
+        fse.rmdirSync(path)
+      }, 10_000)
+      return fse.removeSync(path)
```

Comment:
The timeout is not necessary. Simply call fse.removeSync before, then call fse.rmdirSync. In fact, those two calls are synchronous, so it doesn't even return a Promise (the signature of the method is not correct)

I would suggest something like
```js
await fse.remove(dirPath)
return fse.rmdir(path)
```

Reply:
Actually, it wasn't related to being synchronous or not, `removeSync()` and `rmdirSync()` don't work, unless we wait for some time then try again.

Thanks for your hint, `remove()` simply works!